math/mathcore/src/TMath.cxx

```diff
@@ -550,12 +550,8 @@ Double_t TMath::Normalize(Double_t v[3])
 ////////////////////////////////////////////////////////////////////////////////
 /// Compute the Poisson distribution function for (x,par)
```

Comment:
```suggestion
/// Compute the Poisson distribution function for (x,par).
```
If the sentence isn't ended here, doxygen cannot generate the brief documentation for the overview page.